src/components/player.js

```diff
@@ -174,6 +178,22 @@ export default class Player extends BaseObject {
     }
   }
 
+  registerOptionEventListeners() {
```

Comment:
I'd rather explicitly to use a mapping of events than this, it might let the code clear.

```javascript
var eventsMapping = {
  "onPlay": Events.PLAYER_PLAY,
  "onStop": Events.PLAYER_STOP,
}
var userEvents = this.options.events || {}

Object.keys(userEvents).forEach((userEvent) => {
  this.listenTo(eventsMapping[userEvent], userEvents[userEvent])
})
```